tests/unit/parts/plugins/test_kernel.py

```diff
@@ -1919,7 +1919,7 @@ def _is_sub_array(array, sub_array):
         # but snapd expects modules/ and firmware/
         mv ${CRAFT_PART_INSTALL}/lib/modules ${CRAFT_PART_INSTALL}/
         # remove symlinks modules/*/build and modules/*/source
-        rm ${CRAFT_PART_INSTALL}/modules/*/build ${CRAFT_PART_INSTALL}/modules/*/source
+        rm -r ${CRAFT_PART_INSTALL}/modules/*/build ${CRAFT_PART_INSTALL}/modules/*/source
```

Comment:
```suggestion
        rm -f ${CRAFT_PART_INSTALL}/modules/*/build ${CRAFT_PART_INSTALL}/modules/*/source
```

Reply:
Hi @dariuszd21 , thanks for that, I've fixed it

Reply:
Looks good to me